apps/meteor/app/emoji/client/helpers.ts

```diff
@@ -149,7 +149,10 @@ export const removeFromRecent = (emoji: string, recentEmojis: string[], setRecen
 	setRecentEmojis(recentEmojis);
 };
 
-export const updateRecent = (recentList: string[]) => {
+export const updateRecent = (recentList: string | string[]) => {
+	if (!Array.isArray(recentList)) {
```

Comment:
This fixes: `The client threw an error on updating the emoji list in memory after receiving the updates from the server. `

<img width="701" alt="Screenshot 2024-07-12 at 2 26 20 PM" src="https://github.com/user-attachments/assets/c3e3df51-3b44-4d4c-bbde-7cb5ad770c67">